### PR TITLE
Add description of teams and tech stack

### PR DESCRIPTION
**Pull request by andychisholm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,94 @@
...
+
+## Python
+
+Working in the machine learning domain we've adopted `python` as our lingua-franca.
+We are at one with the [zen](https://peps.python.org/pep-0020/#the-zen-of-python) of `python3.8` and think 
+[PEP-484](https://peps.python.org/pep-0484/) (type-hints) was the best thing since sliced bread.
+
+### Toolchain
+
+Python has many limitations outa-the-box.
```
Comment (xvr-hlt) [inline README.md:31]: I'd just change to 'out of the box' :D (fun police)

File: README.md [hunk 2]
```diff
@@ -3,3 +3,94 @@
...
+
+We like `dataclasses` and especially `pydantic` for data structures and configuration. 
+
+#### Machine learning
+
+We sit precariously on both sides of the `pytorch` vs. `tensorflow` debate and have found a way to love both. That being said we expect to `import pytorch` and specifically `pytorch-lightning` more often going forward.
+
+We manage hierarchical model configuration with `hydra`.
+
+Other key packages include `numpy`, `einops`, `boto3`, `scikit-learn`, `click`, `fire`, `matplotlib`, `opencv`, `pandas`, `plotly`, `pydicom` and `dunamai`.
```
Comment (xvr-hlt) [inline README.md:64]: delete click (I will never stop posting this)

Comment (xvr-hlt) [reply README.md:64]: Pillow

Comment (xvr-hlt) [reply README.md:64]: tqdm

File: README.md [hunk 3]
```diff
@@ -3,3 +3,94 @@
...
+- 254 TB Networked Storage with 800Gb Connectivity
+
+We also have 4 legacy DGX workstations that we still use for smaller experiments.
+This cluster talks to our cloud assets via `Direct Connect` with a 10Gpbs pipe.
+
+We schedule jobs on the cluster using `slurm` and `ray`.
+
+## Communications
+
+We run sprints from `JIRA` and land most (non-code) documentation in `Conflence`.
```
Comment (xvr-hlt) [inline README.md:94]: Confluence

File: README.md [hunk 4]
```diff
@@ -3,3 +3,94 @@
...
+Some key stats:
+- 20 petaFLOPS Tensor F32 from 64 80GB A100 GPUs
+- 5 TB GPU Memory with 2 TB/s Memory Bandwidth
+- 2,048 CPU Cores and 16 TB System Memory
+- 254 TB Networked Storage with 800Gb Connectivity
+
+We also have 4 legacy DGX workstations that we still use for smaller experiments.
+This cluster talks to our cloud assets via `Direct Connect` with a 10Gpbs pipe.
+
+We schedule jobs on the cluster using `slurm` and `ray`.
```
Comment (xvr-hlt) [inline README.md:90]: Love the demotion of slurm to a footnote

Comment (xvr-hlt) [reply README.md:90]: (This may be more aspirational than the current role slurm plays though)

**Review by xvr-hlt — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (andychisholm) [reply README.md:90]: Hmm, maybe: "We use slurm as an ssh client"? 😉

File: README.md (hunk 3 above)
Comment (andychisholm) [reply README.md:94]: God's work (no memerino)

Comment (andychisholm) [reply README.md:94]: Ta

File: README.md (hunk 4 above)
Comment (andychisholm) [reply README.md:90]: Oh I guess your (comment) could be interpreted different ways. Feel free to unresolve this one if you think it's worth plugging slurm differently? Perhaps a too little laconic here

**Review by xvr-hlt — APPROVED** [review]

**Pull request merged**